drakrun/drakrun/main.py

```
@@ -338,6 +338,17 @@ def upload_artifacts(self, analysis_uid, outdir, subdir=''):
             elif os.path.isdir(file_path):
                 yield from self.upload_artifacts(analysis_uid, outdir, os.path.join(subdir, fn))
 
+    def build_profile_payload(self) -> Dict[str, LocalResource]:
+        profiles = {}
+
+        for profile in dll_file_list:
+            fpath = os.path.join(PROFILE_DIR, f"{profile.dest}.json")
+            if not os.path.exists(fpath):
+                continue
+            profiles[profile.path] = LocalResource(name=profile.path, path=fpath)
```

This construction is broken at the moment - silent error in karton.